README.md

```diff
@@ -15,11 +15,13 @@ using `babylonnet.sh` or `mainnet.sh` scripts.
 
 ## Build Instructions
 
-Run `nix-build -A tezos-client-static -o tezos-static`
-to build staticically linked `tezos-client` executable.
+Run `nix-build -A tezos-client-{mainnet, babylonnet} -o tezos-client`
+to build staticically linked `tezos-client-mainnet` or `tezos-client-babylonnet`
+executable.
```

Comment:
would be nice to have each command flavor fully spelled out and on a separate line - for easy copy-pasting of actual executable commands